__tests__/components/ChatBotBody/UserCheckboxes.test.tsx

```diff
@@ -0,0 +1,141 @@
+import React from "react";
+import "@testing-library/jest-dom";
+import { render, screen, fireEvent } from "@testing-library/react";
+import { useSettingsContext } from "../../../src/context/SettingsContext";
+import { useStylesContext } from "../../../src/context/StylesContext";
+import { usePathsContext } from "../../../src/context/PathsContext";
+import { useSubmitInputInternal } from "../../../src/hooks/internal/useSubmitInputInternal";
+import UserCheckboxes from "../../../src/components/ChatBotBody/UserCheckboxes/UserCheckboxes";
+
+// Mocking the context and hook
+jest.mock("../../../src/context/SettingsContext", () => ({
+	useSettingsContext: jest.fn(),
+}));
+jest.mock("../../../src/context/StylesContext", () => ({
+	useStylesContext: jest.fn(),
+}));
+jest.mock("../../../src/context/PathsContext", () => ({
+	usePathsContext: jest.fn(),
+}));
+jest.mock("../../../src/hooks/internal/useSubmitInputInternal", () => ({
+	useSubmitInputInternal: jest.fn(),
+}));
+
+describe("UserCheckboxes Component", () => {
+	const mockSettingsContext = {
+		settings: {
+			general: {
+				primaryColor: "#000",
+				actionDisabledIcon: "disabled-icon-url",
+			},
+			chatInput: {
+				sendCheckboxOutput: true,
+			},
+			botBubble: {
+				showAvatar: true,
+			},
+		},
+	};
+
+	const mockStylesContext = {
+		styles: {
+			botCheckboxRowStyle: { borderColor: "#000" },
+			botCheckboxNextStyle: { color: "#000" },
+			botCheckMarkStyle: { color: "transparent" },
+			botCheckMarkSelectedStyle: { backgroundColor: "#000" },
+		},
+	};
+
+	const mockPathsContext = {
+		paths: [],
+	};
+
+	const mockHandleSubmitText = jest.fn();
+
+	beforeEach(() => {
+		(useSubmitInputInternal as jest.Mock).mockReturnValue({ handleSubmitText: mockHandleSubmitText });
+		(useSettingsContext as jest.Mock).mockReturnValue(mockSettingsContext);
+		(useStylesContext as jest.Mock).mockReturnValue(mockStylesContext);
+		(usePathsContext as jest.Mock).mockReturnValue(mockPathsContext);
+	});
+
+	afterEach(() => {
+		jest.clearAllMocks();
+	});
+
+	test("renders checkboxes correctly", () => {
```

Comment:
Let's standardize to use `it` instead of `test`.

Reply:
Hi! I updated it to use "it"